src/neuroglancer/datasource/dvid/backend.ts

```diff
@@ -18,17 +18,32 @@ import {registerChunkSource} from 'neuroglancer/chunk_manager/backend';
 import {TileChunkSourceParameters, TileEncoding, VolumeChunkSourceParameters} from 'neuroglancer/datasource/dvid/base';
 import {ParameterizedVolumeChunkSource, VolumeChunk} from 'neuroglancer/sliceview/backend';
 import {ChunkDecoder} from 'neuroglancer/sliceview/backend_chunk_decoders';
+import {decodeCompressedSegmentationChunk} from 'neuroglancer/sliceview/backend_chunk_decoders/compressed_segmentation';
 import {decodeJpegChunk} from 'neuroglancer/sliceview/backend_chunk_decoders/jpeg';
 import {decodeRawChunk} from 'neuroglancer/sliceview/backend_chunk_decoders/raw';
+import {VolumeType} from 'neuroglancer/sliceview/base';
 import {CancellationToken} from 'neuroglancer/util/cancellation';
+import {vec3} from 'neuroglancer/util/geom';
 import {openShardedHttpRequest, sendHttpRequest} from 'neuroglancer/util/http_request';
+import {RPC} from 'neuroglancer/worker_rpc';
 
 const TILE_CHUNK_DECODERS = new Map<TileEncoding, ChunkDecoder>([
   [TileEncoding.JPEG, decodeJpegChunk],
 ]);
 
 @registerChunkSource(VolumeChunkSourceParameters)
 class VolumeChunkSource extends ParameterizedVolumeChunkSource<VolumeChunkSourceParameters> {
+  compressedSegmentationBlockSize: vec3|undefined;
```

Comment:
Instead of adding the volumeType parameter and this extra property, you can reuse the existing compressedSegmentationBlockSize property of VolumeChunkSpecification, available as this.spec, which will let you simplify your code.